### DISCUSSION TITLE
Free access to GitHub copilot for open source maintainer

### DISCUSSION THREAD

**Discussion by zcong1993** [community/community#51188] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am team member of https://github.com/zeromicro, and already have write access of https://github.com/zeromicro/go-zero repo which is one of the most popular Go frameworks. But I still haven't gotten free access to GitHub Copilot, so please let me know if I've missed something.

**Comment by Akash1134** [comment, marked as answer]
Hi @zcong1993 The best place to get details and specifics on your Open Source Maintainership that determines access to Copilot would be the billing team. Please [open a support ticket](https://support.github.com/contact?source=subtitle&tags=rr-general-technical) with them. Thanks!

**Reply by zcong1993** [reply]
Thanks, I'll open a ticket.
